driver-core/src/main/java/com/datastax/driver/core/Host.java

```diff
@@ -221,6 +241,32 @@ public VersionNumber getCassandraVersion() {
     }
 
     /**
+     * The DSE version the host is running.
+     * <p/>
+     * It is also possible for this information to be unavailable. In that case
+     * this method returns {@code null}, and the caller should always be aware of this
+     * possibility.
+     *
+     * @return the DSE version the host is running.
+     */
+    public VersionNumber getDseVersion() {
```

Comment:
It looks like I was wrong about this, the problem was the control host not being set, after that was fixed everything seems to be fine.